packaging_automation/citus_package.py

```diff
@@ -18,15 +20,15 @@
                                   get_supported_postgres_nightly_versions,
                                   get_supported_postgres_release_versions,
                                   platform_names,
-                                  run_with_output, str_array_to_str,
-                                  supported_platforms,
+                                  run_with_output, supported_platforms,
                                   transform_key_into_base64_str)
 from .packaging_warning_handler import validate_output
 
 GPG_KEY_NAME = "packaging@citusdata.com"
 
 POSTGRES_VERSION_FILE = "supported-postgres"
 POSTGRES_MATRIX_FILE_NAME = "postgres-matrix.yml"
+POSTGRES_EXCLUDE_FILE_NAME = "pg_exclude.yml"
```

Comment:
Now we have 3 files, and it is quite hard to differentiate their purposes. Can we move the contents of this file inside `postgres-matrix.yml` instead?

Reply:
Actually supported-postgres is just a auto-generated file from postgres-matrix.yml. So we have just 2 files for users.
Actually we will not need this file,pg_exclude.yml, quite common so I prefer not adding this option into postgres_matrix file. Additionally, postgres matrix includes all versions which is some sort of a citus postgres version support matrix file. This file will be very long and adding additional things will make the file more difficult to read.
However, it is not a bad idea. If it is needed, I need to open a PR about that and merge them in the future since there are more prioritized tasks waiting my attention in my work queue.